src/mpl/configure.ac

```diff
@@ -946,6 +946,14 @@ AC_SUBST(libmpl_so_versionflags)
 ## START OF GPU CODE
 #######################################################################
 
+PAC_SET_HEADER_LIB_PATH([cuda])
+PAC_CHECK_HEADER_LIB([cuda_runtime_api.h],[cudart],[cudaStreamSynchronize],[have_cuda=yes],[have_cuda=no])
+if test "${have_cuda}" = "yes" ; then
+    AC_DEFINE([MPL_HAVE_CUDA],[1],[Define is CUDA is available])
+    AC_DEFINE([MPL_HAVE_GPU],[1],[Define if GPU is available])
+fi
```

Comment:
This adds CUDA support, not ZE support as the commit message suggests.

Reply:
fixed the commit message

Reply:
I missed `Define is CUDA is available` -> `Define if CUDA is available`

Reply:
fixed in the latest push.